src/test/resources/features/EndToEnd.feature

```diff
@@ -91,7 +91,18 @@ Feature: HOCS is able to move cases through the entire flow
    #   | DCU TRO |
    #   | DCU N10 |
 
-  #Remember to do Copy to #10
+  @EndToEnd @DCUMIN @Critical
+  Scenario: End to End flow and send a copy to N10
```

Comment:
ideally should only have 1 when and 1 then, if you have a lot of setup just have it as part of your given using * or and
eg:
```java
Scenario: Unlock Document
    Given test
    And test 
    And test
    When test 
    Then test
```
or
```java
Scenario: Unlock Document
    Given test
    * test
    * test
    When test
    Then test
```